symfony/plugins/orangehrmMaintenancePlugin/config/gdpr_access_employee_strategy.yml

```diff
@@ -296,48 +292,49 @@ Entities :
 #          - {field: "achievement" ,class: 'FormatWithAchievement'}
 #          - {field: "added_date" }
 #          - {field: "modified_date" }
-#
-#  LeaveRequestComment:
-#    AccessStrategy:
-#      Basic:
-#        match_by:
-#          - {match: emp_number, join: LeaveRequest}
-#        parameters:
-#          - {field: "created" }
-#          - {field: "created_by_name" }
-#          - {field: "comments" }
-#
+
+  LeaveRequestComment:
+    AccessStrategy:
+      Basic:
+        match_by:
+          - {match: createdByEmployee, join: leaveRequest}
+        parameters:
+          - {field: "createdAt", class: 'FormatWithDate'}
+          - {field: "createdBy"}
+          - {field: "comment" }
+
   LeaveRequest:
     AccessStrategy:
       Basic:
         match_by:
           - {match: employee}
         parameters:
-          - {field: "leaveType" }
-          - {field: "dateApplied" }
-#          - {field: "comments" }
-#
-#  LeaveComment:
-#    AccessStrategy:
-#      Basic:
-#        match_by:
-#          - {match: emp_number, join: Leave}
-#        parameters:
-#          - {field: "created_by_name" }
-#          - {field: "created" }
+          - {field: "leaveType" , getter : ['getLeaveType','getName'] }
+          - {field: "dateApplied" ,class : 'FormatWithDate'}
 #          - {field: "comments" }
+
+  LeaveComment:
+    AccessStrategy:
+      Basic:
+        match_by:
+          - {match: createdByEmployee, join: Leave}
+        parameters:
+          - {field: "createdBy" }
+          - {field: "createdByEmployee" }
+          - {field: "comment" }
+
   Leave:
     AccessStrategy:
       Basic:
         match_by:
           - {match: employee}
         parameters:
-          - {field: "date" }
+          - {field: "date", class : 'FormatWithDate'}
           - {field: "lengthHours" }
           - {field: "lengthDays" }
 #          - {field: "comments" }

```

Comment:
@devishke-orange we can remove this right?

Reply:
yes `comments` field is not there in `Leave` entity